components/bl-leaflet-map/src/helpers/map.js

```diff
@@ -132,19 +132,49 @@ function clearOldMarkers(markersArray) {
   }
 }
 
+function validatePolygon(polygon) {
+  const MINIMAL_POLYGON_LENGTH = 2;
+  const isDescriptionValid = polygon.description === undefined || typeof polygon.description === 'string';
+  const hasPolygon = polygon.polygon?.boundary?.points?.length > MINIMAL_POLYGON_LENGTH;
```

Comment:
polygon has polygon

Sounds not pretty clear. How about "hasMinimalPointsCount" or something that fit this condition? Also minimal polygon length is 3. And condition here should be `>= 3`